src/main/java/de/butbetter/keksys/Calculator.java

```diff
@@ -13,6 +13,7 @@
 	 * @param input input value
 	 * @param percent percentage to add
 	 * @return value plus percentage of that value
+	 * @throws IllegalArgumentException thrown, if calculated value is bigger than float allows
 	 */
-	Float calculateVAT(float input, float percent);
+	Float calculateVAT(float input, float percent) throws IllegalArgumentException;
```

Comment:
Since the input comes from the controller, this needs to be set for the controller as well, which it's not, therefore the Interface and implementation will stay the same, till we set a Task for this

Reply:
Added Task for this issue in jira